golang/Makefile

```diff
@@ -40,20 +40,28 @@ endif
 
 GOPARAMS:=GOOS=$(GOOS) CGO_ENABLED=0 $(GRPC_DEBUG_FLAGS)
 
+# cli start fully containerized stack, stable branch
```

Comment:
`down` has the `--prefix dyo-stable`, doesn't the `up` command need it too?

Reply:
That is the default in case of up, same with down, it is not necessary.